src/main/resources/messages-internal.yml

```diff
@@ -70,6 +70,10 @@ en:
   command_backup: creates configuration and database backup. Use only from console on empty server!
   command_globaltags: displays global tags
   command_globalpoints: displays global points
+  command_download: download quests and templates from a github repository
   offline: '&4All players must be offline in order to make a backup!'
   default_language_changed: '&2Default language changed!'
   error: '&cThere has been an error: {1}'
+  download_scheduled: '&bInitiated download...'
+  download_success: '&aDownload completed!'
+  download_failed: "&4Download failed due to the following error:\n&c{1}\n&7&osee log for more details"
```

Comment:
I dont like this formatting here. Especially the `\n&c{1}\n&7&osee log for more details"` part. May we should discuss a better error formatting in the TeamSpeak